doc/build/orm/collections.rst

```diff
@@ -203,24 +204,28 @@ to achieve a simple dictionary collection.  It produces a dictionary class that
 of the mapped class as a key.   Below we map an ``Item`` class containing
 a dictionary of ``Note`` items keyed to the ``Note.keyword`` attribute::
 
-    from sqlalchemy import Column, Integer, String, ForeignKey
+    from sqlalchemy import Column, ForeignKey, Integer, String
+    from sqlalchemy.ext.declarative import declarative_base
```

Comment:
this should be sqlalchemy.orm.declarative_base

these should all have been migrated awhile back so not sure how this is still around (others maybe, I guess, if this one was missed)

Reply:
I didn't really pay attention to imports, since declarative is now part of orm i will fix this